src/libstd/sys/common/net.rs

```diff
@@ -167,7 +167,7 @@ impl TcpStream {
         let sock = Socket::new(addr, c::SOCK_STREAM)?;
 
         let (addrp, len) = addr.into_inner();
-        cvt_r(|| unsafe { c::connect(*sock.as_inner(), addrp, len) })?;
+        cvt(unsafe { c::connect(*sock.as_inner(), addrp, len) })?;
```

Comment:
Why are these switching away from `cvt_r`? They're supposed to handle `EINTR` internally and this change means that they no longer handle that.